atm.js

```diff
@@ -3,19 +3,49 @@ checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
 var checkingDeposit = function(amount){
   checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+};
+
+  var checkingDeposit = {
+    var account: $("#account");
```

Comment:
remove semi-colon. I also recommend keeping the console open (cmd-option-j) while you test out. the semi-colon will cause a syntax error